helix-term/src/commands.rs

```diff
@@ -5115,11 +5115,25 @@ fn surround_replace(cx: &mut Context) {
             }
         };
 
+        // Visual feedback
+        let selection = selection.clone();
+        let mut ranges: SmallVec<[Range; 1]> = SmallVec::new();
+        change_pos.chunks(2).for_each(|p| {
+            if p.len() == 2 {
```

Comment:
With `chunks_exact` you can then avoid this `len` check

I'd also leave a note to use `array_chunks` in the future once stabilized